apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -372,10 +372,35 @@ private class CredentialWrapper : ITokenAccess
         }
 
         [Fact]
+        public async Task CommandTimeout()
+        {
+            long result = 0;
+            var e = await Assert.ThrowsAsync<SpannerException>(
+                async () =>
+                {
+                    using (await _testFixture.GetTestDatabaseConnectionAsync())
+                    {
+                        using (var connection =
+                            new SpannerConnection($"{_testFixture.ConnectionString};AllowZeroTimeouts=true"))
```

Comment:
Ooh, I hadn't thought of making it part of the connection string. That's neat. Not sure about the name though, as a timeout of 0 is "allowed" anyway - it just means "infinite".

How about AllowImmediateTimeouts?